MQTicketService/Dockerfile

```diff
@@ -47,10 +46,11 @@ USER 0
 #
 #
 RUN if [ "$platformArch" == "amd64" ]; then \
```

Comment:
`platformArch` is overloaded as both an ARG and an ENV, can they be individually named? e.g., `argPlatformArch` and `envPlatformArch`